### PR TITLE
[docs] - Update ECS deployment guide

### PR DESCRIPTION
**Pull request by erinkcochran87**
This PR updates the **Deploying to AWS ECS** guide with some copy editing and formatting/style changes. No content in here has changed - just formatting and editing for clarity/style.

### CODE REVIEW

File: docs/content/deployment/guides/aws.mdx [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
-title: Deploying Dagster to AWS | Dagster
+title: Deploying Dagster to Amazon Web Services | Dagster
 description: To deploy Dagster to AWS, EC2 or ECS can host Dagit, RDS can store runs and events, and S3 can act as an IO manager.
 ---
 
-# Deploying Dagster to AWS
+# Deploying Dagster to Amazon Web Services
 
-This guide provides instructions for deploying Dagster on AWS. You can use EC2 or ECS to host Dagit and the Dagster Daemon, RDS to store runs and events, and S3 as an IO manager to store op inputs and outputs.
+This guide provides instructions for deploying Dagster on Amazon Web Services (AWS). You can use EC2 or ECS to host Dagit and the Dagster daemon, RDS to store runs and events, and S3 as an I/O manager to store op inputs and outputs.
```
Comment (gibsondan) [inline docs/content/deployment/guides/aws.mdx:8]: Maybe "You can use EC2 or ECS to host Dagit and the Dagster daemon" => You can use EC2 or ECS to host Dagster"? "Dagit and the Dagster daemon" leaves out grpc servers and runs

File: docs/content/deployment/guides/aws.mdx [hunk 2]
```diff
@@ -26,25 +30,37 @@ storage:
...
-In this case, you'll want to ensure you provide the right connection strings for your RDS instance, and ensure that the node or container hosting Dagit is able to connect to RDS.
+In this case, you'll want to ensure that:
+
+- You provide the right connection strings for your RDS instance
+- The node or container hosting Dagit is able to connect to RDS
+
+Be sure that this file is present and `DAGSTER_HOME` is set on the node where Dagit is running.
 
-Be sure that this file is present, and _DAGSTER_HOME_ is set, on the node where Dagit is running.
+**Note**: Using RDS for run and event log storage doesn't require that Dagit be running in the cloud. If you're' connecting a local Dagit instance to a remote RDS storage, verify that your local node is able to connect to RDS.
```
Comment (gibsondan) [inline docs/content/deployment/guides/aws.mdx:40]: stray apostrophe after you're

File: docs/content/deployment/guides/aws.mdx [hunk 3]
```diff
@@ -26,25 +30,37 @@ storage:
...
+The Deploying on ECS example on GitHub demonstrates how to configure the [Docker Compose CLI integration with ECS](https://docs.docker.com/cloud/ecs-integration/) to manage all of the required AWS resources that Dagster needs to run on ECS. The example includes:
+
+- A Dagit container for loading and launching jobs
+- A `dagster-daemon` container for managing a run queue and submitting runs from schedules and sensors
+- A Postgres container for persistent storage
+- A container with user job code. The [Dagster instance](/deployment/dagster-instance) uses the <PyObject module="dagster_aws.ecs" object="EcsRunLauncher" /> to launch each run in its own ECS task.
```
Comment (gibsondan) [inline docs/content/deployment/guides/aws.mdx:53]: I never know what to call the thing being called "user job code" here :)

Comment (gibsondan) [inline docs/content/deployment/guides/aws.mdx:53]: The sentence about the run launcher isn't directly related to the sentence before it - probably should live outside of the bullet list

File: docs/content/deployment/guides/aws.mdx [hunk 4]
```diff
@@ -26,25 +30,37 @@ storage:
...
 
-### Launching Runs in ECS
+### Launching runs in ECS
 
 The <PyObject module="dagster_aws.ecs" object="EcsRunLauncher" /> launches an ECS task per run. It assumes that the rest of your Dagster deployment is also running in ECS.
 
-By default, each run's task registers its own task definition. To simplify configuration, these task definitions inherit most of their configuration (networking, cpu, memory, environment, etc.) from the process that launches the run but overrides its container definition with a new command to launch a Dagster run. When using the <PyObject module="dagster._core.run_coordinator" object="DefaultRunCoordinator" />, runs launched via Dagit or GraphQL inherit their task definitions from the Dagit task; runs launched from a sensor or schedule inherit their task definitions from the Daemon task.
+By default, each run's task registers its own task definition. To simplify configuration, these task definitions inherit most of their configuration (networking, cpu, memory, environment, etc.) from the process that launches the run but overrides its container definition with a new command to launch a Dagster run.
```
Comment (gibsondan) [inline docs/content/deployment/guides/aws.mdx:59]: not new, but "process that launches the run" should probably be "task that launches the run"

File: docs/content/deployment/guides/aws.mdx [hunk 5]
```diff
@@ -26,25 +30,37 @@ storage:
...
+
+When using the <PyObject module="dagster._core.run_coordinator" object="DefaultRunCoordinator" />, runs launched via Dagit or GraphQL inherit their task definitions from the Dagit task while runs launched from a sensor or schedule inherit their task definitions from the daemon task.
```
Comment (gibsondan) [inline docs/content/deployment/guides/aws.mdx:61]: not directly related but just flagging that throughout the docs the way we talk about the run queue is pretty confusing, dunno why a user should have to be aware of a specific class name like DefaultRunCoordinator

File: docs/content/deployment/guides/aws.mdx [hunk 6]
```diff
@@ -170,6 +186,6 @@ resources:
       s3_prefix: good/prefix-for-files-
 ```
 
-The resource uses `boto` under the hood, so if you are accessing your private buckets, you will need to provide the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables or follow one of the other boto authentication methods.
+The resource uses `boto` under the hood. If you're accessing your private buckets, you will need to provide the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables or follow one of the other boto authentication methods.
```
Comment (gibsondan) [inline docs/content/deployment/guides/aws.mdx:189]: we could link to https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html#configuring-credentials in the last bit

**Review by gibsondan — APPROVED** [review]

File: docs/content/deployment/guides/aws.mdx (hunk 1 above)
Comment (ei-grad) [reply docs/content/deployment/guides/aws.mdx:8]: "to host Dagster components" maybe?

**Pull request merged**